os/net/ipv6/uip-ds6-route.c

```diff
@@ -310,7 +310,7 @@ uip_ds6_route_lookup(const uip_ipaddr_t *addr)
     LOG_INFO_6ADDR(uip_ds6_route_nexthop(found_route));
     LOG_INFO_("\n");
   } else {
-    LOG_WARN("No route found\n");
+    LOG_INFO("No route found\n");
```

Comment:
From what I understand, taking this path means using the default route. Is that something the user should be worried about, or should the message include ", using default route"?

Reply:
Although this function is indeed used for routing decisions (where the default route would be used if route not found), the function is a generic lookup and is also for other purposes - so it would be inappropriate to add a "using default route" here. For routing decisions there is actual such a log (at INFO level) in tcpip.c:529.

I cannot see the need for WARN-level for the lookup not finding a route. It makes the WARN-level verbose since it is printed every time the default route is used - which imo is not something for the user to be worried about.

Reply:
Agree with Andreas here